src/test/scala/nl.knaw.dans.easy.deposit/DepositDirSpec.scala

```diff
@@ -53,13 +53,13 @@ class DepositDirSpec extends TestSupportFixture with MockFactory {
         val dir = draftsDir / "user001" / d.id.toString
         dir should exist
         (dir / "deposit.properties") should exist
-        (dir / "bag") should exist
-        (dir / "bag/bag-info.txt") should exist
-        (dir / "bag/bagit.txt") should exist
-        (dir / "bag/manifest-sha1.txt") should exist
-        (dir / "bag/tagmanifest-sha1.txt") should exist
-        (dir / "bag/data") should exist
-        (dir / "bag/metadata") should exist
+        (dir / bagDirName) should exist
+        (dir / s"$bagDirName/bag-info.txt") should exist
```

Comment:
while you're at it, make these like:

```suggestion
        (dir / bagDirName / "bag-info.txt") should exist
```